### PR TITLE
feat: start new design

### PR DESCRIPTION
**Pull request by DarinMao**
https://www.figma.com/file/62ByqrNpfWiNRXeBJQbZmH/TJCSC?node-id=0%3A1

https://github.com/TJCSec/website/projects/1#card-51762432
https://github.com/TJCSec/website/projects/1#card-51758981

### CODE REVIEW

File: src/components/hero.js [hunk 1]
```diff
@@ -1,30 +1,42 @@
 /** @jsx jsx */
 import { Box, jsx } from 'theme-ui'
 
-const Hero = ({ children, big, ...props }) => (
+import CircuitBoard from '../images/circuit-board.svg'
```
Comment (AutinMitra) [inline src/components/hero.js]: We may want to setup something like absolute paths so we don't have to go through a bunch of nested relative directories for importing

Comment (DarinMao) [reply src/components/hero.js]: meh, one .. is not too bad I think

**Comment by DarinMao** [comment]
subtitle is a little large

File: src/components/hero.js (hunk 1 above)
Comment (AutinMitra) [reply src/components/hero.js]: yeah definitely not bad at all lol, not sure how the routing works for Gatsby but in nextjs it was all nested dirs

**Comment by AutinMitra** [comment]
Maybe slightly increase margin between items on the right side of the navbar

**Comment by AutinMitra** [comment]
I think one final minor change is maybe decrease the padding on the mobile - text is dangerously close to overflowing on iPhone-6 sized device, and will probably overflow on an iPhone 5 or smaller (not that anyone uses em)

**Comment by AutinMitra** [comment]
Otherwise, I think the homescreen follows the mockup nicely

**Comment by DarinMao** [comment]
I think there should be some additional content about the club as a whole (officers, what we do, etc), and CTA can just scroll down

**Comment by AutinMitra** [comment]
Yeah we can have another section within the home route for club-info/QA/officers, I'll make sure to add it in the figma tomorrow

**Review by AutinMitra — APPROVED** [review]
Seems good for now!

**Comment by AutinMitra** [comment]
Seems all good to me

**Review by AutinMitra — APPROVED** [review]

**Pull request merged**